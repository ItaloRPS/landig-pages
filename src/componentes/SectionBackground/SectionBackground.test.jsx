import { screen } from '@testing-library/jest-dom';
import {renderTheme} from '../../styles/render-theme'
import {SectionBackground} from '.'
describe('<SectionBackground/>',()=>{
    test('shold render with background dark',()=>{
        const  {container}=  renderTheme(<SectionBackground>Children</SectionBackground>);
        expect(container).toMatchSnapshot()
    });

    test('shold render with background light',()=>{
        const  {container}=  renderTheme(<SectionBackground>Children</SectionBackground>);
        expect(container).toMatchSnapshot()
    });
});