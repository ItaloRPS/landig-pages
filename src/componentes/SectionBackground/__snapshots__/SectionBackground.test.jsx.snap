// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionBackground/> shold render with background dark 1`] = `
.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 5rem;
}

<div>
  <div
    class="c0"
  >
    Children
  </div>
</div>
`;

exports[`<SectionBackground/> shold render with background light 1`] = `
.c0 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 5rem;
}

<div>
  <div
    class="c0"
  >
    Children
  </div>
</div>
`;
